components/public/sites/FindStoreModal.js

```diff
@@ -32,6 +32,9 @@ const FindStoreModal = (props) => {
           .then((body) => {
             const { response } = body;
             setSites(response.data.pageContent);
+          })
+          .catch((e) => {
```

Comment:
we want to keep this console.log?

Reply:
I was thinking we keep some of these error console logs for now as a placeholder for the developer who picks up a future ticket for error handling and displaying messages